test/Npgsql.Tests/Support/PgPostmasterMock.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+using Npgsql.Util;
+
+namespace Npgsql.Tests.Support
+{
+    class PgPostmasterMock : IAsyncDisposable
```

Comment:
`PgServerMock` is `public` - maybe do the same here?

Reply:
Will just keep both internal for now, it doesn't really matter as these are test supports types.